### PR TITLE
Hotbar visual improvements

### PR DESCRIPTION
**Pull request by aleksfront**
* Empty cells with ability to add/remove them (max amount is 12)
* Tiny scrollbar and bottom shadow to keep cells accessible on small height
* Cluster `online` status coloring
* Cell click animations regarding it's empty or not.

Adding/removing hotbar cells

https://user-images.githubusercontent.com/9607060/116205141-1cf06d00-a746-11eb-8df6-47749c18ef5b.mp4

Switching between clusters

https://user-images.githubusercontent.com/9607060/116205232-342f5a80-a746-11eb-9067-b1b3872f2490.mp4

**Note**. Removing file `lens-hotbar-store.json` from `/Users/username/Library/Application Support/OpenLensDev` might needed before to run properly.

### CODE REVIEW

File: src/common/catalog-entities/kubernetes-cluster.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export class KubernetesCluster extends CatalogEntity<CatalogEntityMetadata, Kube
       },
     ];
 
-    if (this.status.active) {
+    if (this.status.phase == "connected") {
```
Comment (jakolehm) [inline src/common/catalog-entities/kubernetes-cluster.ts:53]: Why this change? 🤔

Comment (aleksfront) [reply src/common/catalog-entities/kubernetes-cluster.ts:53]: Mixed concepts. Active not equals to connected, it's something shared between any kind of catalog entities. Connect/Disconnect seems a more proper relation to me.

**Review by jakolehm — APPROVED** [review]

**Review by nevalla — APPROVED** [review]
LGTM
